attempt_6_animals/simple-reference-animals.squiggle

```diff
@@ -126,16 +111,260 @@ improvement_as_proportion_of_lifetime = 0.05 to 0.5
 value_fwi_fish = 
   fish_potentially_helped * 
 	improvement_as_proportion_of_lifetime * 
-	(value_happy_salmon_year_in_human_qalys / salmon_estimates.lifetime)
+	(salmon_estimates.value_year_in_human_qalys / salmon_estimates.lifetime)
 
 value_of_shrimp_in_fish = (0.3 to 1)
 // ^ very uncertain, subjective
 
 value_fwi_shrimp = 
   shrimp_potentially_helped * 
 	improvement_as_proportion_of_lifetime * 
-	(value_happy_salmon_year_in_human_qalys / salmon_estimates.lifetime) *
+	(salmon_estimates.value_year_in_human_qalys / salmon_estimates.lifetime) *
   value_of_shrimp_in_fish
 
 value_fwi_so_far = value_fwi_fish + value_fwi_shrimp
-value_fwi_so_far
+proportion_fwi_in_2022 = 1/4 to 1/2
+value_fwi  = value_fwi_so_far * proportion_fwi_in_2022
+fwi_item = {
+  name: "Fish Welfare Inititative (2022)",
+	value: value_fwi -> ss 
+}
+
+/*
+## Open Wing Alliance
```

Comment:
A bit later on, I could see having the Animal evaluators in a separate file/component, but defining them in terms of a few passed-in variables for the animals.

Reply:
Agree, but probably not yet.